TextAnalyzer/CountEveryLetterReportCommand.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+
+namespace TextAnalyzer
+{
+    class CountEveryLetterReportCommand : ICommand
+    {
+        public string Description => "Generate report for each letter from file";
+
+        public void Activate()
+        {
+            List<LetterOccurrenceModel> letterOccurrenceList = new List<LetterOccurrenceModel>();
+
+            String text = Regex.Replace(TextFileFetcher.GetTextFileString(), @"(\W|\d|_)+", "");
```

Comment:
Ten sam regex jest użyty i tu i w `LettersCountReportCommand.cs`.
Sugeruję wyciągnąć z obu miejsc do nowej statycznej klasy np. `RegexHelper.cs`, nazwa metody która to zastąpi ma też duże szanse być bardziej opisowa niż instrukcja sama w sobie.